cypress/e2e/super-admin/default-tests/launchExternalStudent.cy.js

```diff
@@ -2,14 +2,12 @@ const baseUrl = Cypress.config().baseUrl;
 const testDistrictId = Cypress.env('testDistrictId');
 const testPartnerAdministrationName = Cypress.env('testPartnerAdministrationName');
 const testPartnerAdministrationId = Cypress.env('testPartnerAdministrationId');
-const testPartnerAdminUsername = Cypress.env('PARTNER_ADMIN_USERNAME');
-const testPartnerAdminPassword = Cypress.env('PARTNER_ADMIN_PASSWORD');
 const testUserList = Cypress.env('testUserList');
 
-describe('Partner Admin: Individual Reports', () => {
+describe('Partner Admin: Launching Students', () => {
```

Comment:
```suggestion
describe('Super Admin: Launching Students', () => {
```

Reply:
@lucasxsong this was resolved but no change was made. Since this test suite is part of the super admin tests, I believe the describe block should be as follows? 

```suggestion
describe('Super Admin: Launching Students', () => {
```

Reply:
Sorry about that – I thought I'd added the commit but I'm not sure how it fell through the cracks. I've just updated to reflect the correct naming.